hooks/pre-commit.d/cargo-audit

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+ [[ ! -z "$TRAVIS" && ! "$1" == "--force" ]] && exit 0
+ cargo audit
```

Comment:
This file appears to have stray spaces at the beginning of the lines.

Reply:
D'oh. Sorry -- fixed.